.gitignore

```diff
@@ -1,4 +1,5 @@
 results/
+fitbenchmarking_results/
```

Comment:
with the renaming of result/ -> fitbenchmarking_results/, do we still want to ignore result/ also?